.gitignore

```diff
@@ -3,3 +3,5 @@
 /python_memcached.egg-info
 .tox
 .coverage
+.idea
```

Comment:
Please revert these changes.

Reply:
No way

Reply:
In my experiencing maintaining Django, we've recommended to use `git config core.excludesfile` for excluding IDE-specific files. That way if a new IDE comes along, its users can update their configuration own rather than requiring many projects to update `.gitingore`. I was also suggesting to exclude unrelated changes like this in a bug fix as that can be confusing when looking at `git blame`.

As an aside, this isn't my pet project, I just asked Sean this week if I could help him maintain it as it seems inactive.

Reply:
Well thats great that you have a Collaborator tag and I have nothing even though I've pointed you at several other issues in the past week that you have been actively helping clean up.   I guess if thats the standard way to do it ... I wouldn't mind if the .gitignore got long because it would at least be a way to ignore things we know are common for the community.   I personally use mercurial way more than git, so I don't know all the bells and whistles.   It saddens me that the desired approach is that everyone does a one-off and pretends like the world is IDE agnostic.

Reply:
I'm fine smashing it down to a single commit -- I mean personally I would never do it, because I am against re-writing history, but whatever.  I really just want this bug fixed.   I put 3 hours into this last night just trying to get it this far.  So you figure I have 8 hours into trying to improve this library.

Reply:
I really appreciate both of you collaborating on this and the effort both of you have put into this.  It saddens me that this has devolved into anger.  It would sadden me if you chose to not participate, especially after all your work, because ".idea" is not added to the project .gitignore.  If that is the choice you want to make, I'll support your choice though.

Thank you @timgraham for the pointer about core.excludesfile, I was not aware of that.  I agree that that is the right place for this exclude, after reading the git docs describing it as the preferred solution for files generated by a users editor.  Quoting:

    Patterns which a user wants git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesfile in the user’s ~/.gitconfig

Quoted from: https://git-scm.com/docs/gitignore/1.7.12

Reply:
It is because you are way more unreasonable than working with the DRF creator.  I assume its because he is from Europe and you are not.   The .idea directory will always be the PyCharm/IntelliJ local IDE config for all users, but I assume its because you want no reference to commercial software in your OSS.    You would rather enforce these kind of standards than to actually realize that so many people are running into this problem with python 3 in their production code.  Its not really acceptable when memcached error crashes the page for the end-user, but I guess we can argue about what should live in which .gitconfig.   Either way, you'd have to be stupid to ever want users to add a .idea directory to the root of your project.